test/e2e/common/node/pod_level_resources.go

```diff
@@ -165,7 +165,7 @@ func podLevelResourcesTests(f *framework.Framework) {
 
 	tests := []testCase{
 		{
-			name: "Guaranteed QoS pod with container resources",
+			name: "Guaranteed QoS pod with only container resources",
```

Comment:
I changed this because it was duplicated with the test case name at l.188.
https://github.com/kubernetes/kubernetes/blob/027447e683de1e14c1476670eef5bfa6c965b2d3/test/e2e/common/node/pod_level_resources.go#L188

Reply:
Thank you for fixing this and also for the PR :)